indicators/indicators.py

```diff
@@ -197,6 +194,143 @@ def run_compute_indicators(fp_di, indicators, coord_labels, kwargs={}):
 
 
+def build_attrs(
+    indicator, scenario, model, start_year, end_year, lat_min, lat_max, lon_min, lon_max
+):
+    """Build standardized attribute dictionarys for computed indicator datasets. This function uses lookup tables imported from indicators/luts.py.
+
+    Args:
+        indicator (str): indicator id
+        model (str): model id
+        scenario (str): scenario id
+        start_year (str): first year of dataset
+        end_year (str): last year of dataset
+        lat_min (str): minimum latitude of dataset
+        lat_max (str): maximum latitude of dataset
+        lon_min (str): minimum longitude of dataset
+        lon_max (str): maximum longitude of dataset
+
+    Returns:
+        global_attrs, var_coord_attrs (tuple): tuple of global and variable/coordinate attribute dictionarys
+    """
+    # test units to determine NA value TODO: revisit this if there are any other integer units besides "days"
+    if units_lu[indicator] != "d":
+        fill_value = "NaN"
+    else:
+        fill_value = "-9999"
+
+    # build global attribute dict for the whole dataset:
+    global_attrs = {
+        "title": f"{indicator_lu[indicator]['title']}, {start_year}-{end_year}: {model}-{scenario}",
+        "author": "Scenarios Network for Alaska and Arctic Planning (SNAP), International Arctic Research Center, University of Alaska Fairbanks",
+        "creation_date": datetime.datetime.now().strftime("%m/%d/%Y, %H:%M:%S"),
+        "email": "uaf-snap-data-tools@alaska.edu",
+        "website": "https://uaf-snap.org/",
+        "references": f"{indicator_lu[indicator]['references']}",  # TODO: add reference information!
+    }
+
+    # but attribute dict for individual coordinates and variables:
+    var_coord_attrs = {
+        "lat": {
+            "name": "latitude",
+            "units": "degrees north",
+            "fill_value": "NaN",
+            "lat_max": f"{lat_max}",
+            "lat_min": f"{lat_min}",
+        },
+        "lon": {
+            "name": "longitude",
+            "units": "degrees east",
+            "fill_value": "NaN",
+            "lon_max": f"{lon_max}",
+            "lon_min": f"{lon_min}",
+        },
+        "year": {
+            "start_year": start_year,
+            "end_year": end_year,
+        },
+        "scenario": {
+            "id": f"{scenario}",
+            "ssp": f"{scenario_lu[scenario]['ssp']}",
+            "forcing_level": f"{scenario_lu[scenario]['forcing_level']}",
+        },
+        "model": {
+            "model": f"{model}",
+            "institution": f"{model_lu[model]['institution']}",
+            "institution_name": f"{model_lu[model]['institution_name']}",
+        },
+        indicator: {
+            "long_name": indicator_lu[indicator]["long_name"],
+            "units": f"{units_lu[indicator]}",
+            "fill_value": fill_value,
+            "description": indicator_lu[indicator]["description"],
+        },
+    }
+    return global_attrs, var_coord_attrs
```

Comment:
```suggestion
    return global_attrs, var_coord_attrs, fill_value
```

Just returning the `fill_value` here since it is correct, we will use it in the replace attrs function